src/assets/sql/stored-procedure/countAppointmentWithUserIdAndServiceIdAndPatientId.sql

```diff
@@ -0,0 +1,69 @@
+-------------------------------------------------------------------------- STORED PROCEDURE SCRIPT----------------------------------------------------------------
+DELIMITER $$
+DROP PROCEDURE IF EXISTS `countAppointmentsByCondition` $$
+CREATE DEFINER=`root`@`localhost` PROCEDURE `countAppointmentsByCondition`(IN `idUserIntegrated` VARCHAR(200), 
```

Comment:
is procedure created at DB level if so EA DB user has to be used instead 'root' user

Reply:
that line is redundant, I already removed it.